frontend/src/components/Button.tsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react'
+import styled from 'styled-components'
+
+const Border = styled.button<{ state: string }>`
+  height: 2.2rem;
+  width: 10vw;
```

Comment:
if the button text is too long, the text will be cut off

Reply:
Communicated with designers and will leave it as it is first since likelihood of this happening is very low